.github/workflows/integration.yml

```
@@ -12,14 +12,14 @@ jobs:
       - uses: rokroskar/workflow-run-cleanup-action@master
         env:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/master'"
+    if: ${{ !startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/master' }}
 
   ibc-integration:
     runs-on: ubuntu-latest
     env:
       COMETBFT_VERSION: 0.37.1
       GAIA_VERSION: 6.0.0
```

Dockerfile and workflow files should be in sync.

```suggestion
      GAIA_VERSION: 7.1.1
```